src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1491,14 +1488,36 @@ internal IEnumerable<PackageManagerSearchElementViewModel> Search(string searchT
         /// <returns></returns>
         private PackageManagerSearchElementViewModel GetViewModelForPackageSearchElement(string packageName)
         {
-            var result = SearchResults.Where(e => e.Name.Equals(packageName));
+            var result = PackageManagerClientViewModel.CachedPackageList.Where(e =>
+            {
+                if (e.Name.Equals(packageName))
+                {
+                    return true;
+                }
+                return false;
```

Comment:
Do we need this if condition? I thought we can just do `CachedPackageList.Where(e =>e.Name.Equals(packageName)`

Reply:
Yep, that's what I thought when I looked at it. But I did not add this code, I just reverted this change to its previous state, thinking don't fix something that ain't broke. ;)

Reply:
updated